src/genmsg/msg_loader.py

```diff
@@ -55,7 +55,12 @@
 from . srvs import SrvSpec
 
 class MsgNotFound(Exception):
-    pass
+
+    def __init__(self, message, base_type, package, search_path):
```

Comment:
Existing code might use the constructor of this class with a single message argument. That needs to continue to work. Therefore please make the new arguments optional.